.travis.yml

```diff
@@ -62,8 +62,9 @@ env:
     - "TESTLANG=Nim"
     - "TESTLANG=Perl"
     - 'TESTDIR="PHP/php"'
+    - 'TESTDIR="PHP/comet"'
```

Comment:
Don't create a new job for only 1 framework.
Please, revert that change.

Reply:
Done